OutaTime/OutaTime/TimeCurcuitViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  TimeCurcuitViewController.swift
+//  OutaTime
+//
+//  Created by Chris Gonzales on 1/29/20.
+//  Copyright © 2020 Chris Gonzales. All rights reserved.
+//
+
+import UIKit
+
+class TimeCurcuitViewController: UIViewController {
+    
+    var speed: Int = 0
+    var timer: Timer? = nil
```

Comment:
Already set the timer var to type Optional Timer. No need to set it equal to nil.